.github/workflows/ci.yml

```diff
@@ -0,0 +1,39 @@
+name: Python CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
+        python-version: ['3.6', '3.7', '3.8']
+        toxenv: ['py']
+        include:
+          - python-version: "3.8"
+            toxenv: "quality"
+            os: [ubuntu-20.04]
+
+    steps:
+    - name: checkout repo
+      uses: actions/checkout@v2
+    - name: setup python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install pip
+      run: pip install -r requirements/pip.txt
+
+    - name: Install Dependencies
+      run: pip install -r requirements/travis.txt
```

Comment:
we should rename the requirement files to say `ci.txt` instead of `travis.txt`

Reply:
I was thinking of doing that later on in the PR along with `.travis` removal.